builddocs.cmd

```diff
@@ -1 +0,0 @@
-@"%~dp0build" docs
```

Comment:
Documentation building needs to be restored as there was no agreement to dropping it as part of targeting .NET Core.